packages/opentelemetry-instrumentation-bedrock/tests/traces/test_anthropic.py

```
@@ -1075,3 +1107,57 @@ def assert_message_in_logs(log: LogData, event_name: str, expected_content: dict
     else:
         assert log.log_record.body
         assert dict(log.log_record.body) == expected_content
+
+
+@pytest.mark.vcr
+def test_anthropic_converse_with_caching(instrument_legacy, brt, span_exporter, reader):
+    response_write = brt.converse(
+    modelId="anthropic.claude-3-haiku-20240307-v1:0",
```

Flake8 (7.3.0)

[error] continuation line missing indentation or outdented

(E122)